src/internal/svg/appleSvg.tsx

```diff
@@ -1,29 +1,24 @@
+import { icon } from '../../styles/theme';
+
 export const appleSvg = (
   <svg
+    data-testid="ock-appleSvg"
+    role="img"
+    aria-label="ock-appleSvg"
+    width="14"
+    height="16"
+    viewBox="0 0 14 16"
+    fill="none"
     xmlns="http://www.w3.org/2000/svg"
-    viewBox="0 -29.75 165.5 165.5"
-    preserveAspectRatio="xMidYMid meet"
-    id="Artwork"
-    data-testid="appleSvg"
-    width="24"
-    height="24"
   >
-    <title>Apple Pay Onramp</title>
+    <title>Apple SVG</title>
```

Comment:
same comment about titles. prefer to remove SVG and make as intuitive as possible when it's read out loud